recipes/intelligent_queue_management/README.md

```diff
@@ -104,4 +104,33 @@ _NOTE: Alternatively, you can run all steps with the following command:_
 python main.py --stream sample_video.mp4
 ```
 
-Congratulations! You have successfully set up and run the Intelligent Queue Management application with OpenVINO™.
+## Step 4
+
+Benchmark the Model with OpenVINO's Benchmark_App
+Benchmarking provides insight into your YOLOv8 model's real-world performance. Performance may vary based on use and configuration.
+
+### Benchmark Results 
+
+![YOLOv8m Benchmark Results](https://github.com/AnishaUdayakumar/intelligent-queue-management-openvino/assets/109281183/8a81243e-ee32-4b30-9994-326ecb07d32f)
```

Comment:
These images must belong to the OpenVINO repository (if possible) and should be separate from each other.

Reply:
I will add the images separately to the OpenVINO repository. Will work on this change.
Can you help me with some leads to add the data to the OpenVINO repository?